cas-server-documentation/installation/SPNEGO-Authentication.md

```diff
@@ -38,19 +38,49 @@ SPNEGO support is enabled by including the following dependency in the WAR overl
 </dependency>
 ```
 
+If maven complains about missing `jcifs-ext` component, you may have to add a repository in the WAR overlay:
+
+```xml
```

Comment:
Please remove this block. We should add this to the default overlay anyways. The template should handle it for all; users should not have to learn this.